lib/offline/storage.js

```diff
@@ -538,19 +554,18 @@ shaka.offline.Storage = class {
    * changes of the other.
    *
    * @param {number} manifestId
-   * @param {!shaka.media.SegmentReference} ref
-   * @param {shaka.extern.SegmentDataDB} data
+   * @param {!Object.<string, number>} manifestUpdates
+   * @param {number} dataSizeUpdate
    * @param {function()} throwIfAbortedFn  A function that should throw if the
    *   download has been aborted.
    * @return {!Promise.<?shaka.extern.ManifestDB>}
    */
-  static async assignStreamToManifest(manifestId, ref, data, throwIfAbortedFn) {
+  static async assignStreamToManifest(
```

Comment:
Nit: The method should probably be called `assignStreamsToManifest` now that it can do multiple at once.

Reply:
Renamed to assignSegmentsToManifest, since it is now only setting the database keys of the segments.  The streams are already part of the manifest.